website/client-old/js/services/guideServices.js

```diff
@@ -82,11 +82,11 @@ function($rootScope, User, $timeout, $state, Analytics) {
         }, {
           element: ".meter.mana",
           title: window.env.t('spells'),
-          content: window.env.t('spellsText') + " <a target='_blank' href='http://habitica.wikia.com/wiki/Todos'>" + window.env.t('toDo') + "</a>."
+          content: window.env.t('spellsText')
         }, {
           orphan: true,
           title: window.env.t('readMore'),
-          content: window.env.t('moreClass') + " <a href='http://habitica.wikia.com/wiki/Class_System' target='_blank'>Wikia</a>.",
+          content: window.env.t('moreClass'),
           final: true
```

Comment:
@MathWhiz That's a good idea. I'll test that tonight if I can.

And oh this is annoying: all my pointless "good" comments were included. I was writing those just as a note to myself to keep track of which bits I'd tested and I thought I'd deleted them before saving my final comments. Sorry about the spam!!